octoprint_PrintTimeGenius/analyzers/analyze_gcode_comments.py

```diff
@@ -61,10 +62,15 @@ def registrar(func):
 def process_slic3r_filament(gcode_line):
   """Match a slic3r PE filament line"""
   ret = dd()
-  m = re.match('\s*;\s*filament used\s*=\s*([0-9.]+)\s*mm\s*\(([0-9.]+)cm3\)\s*', gcode_line)
+  m = re.match('\s*;\s*filament used\s*=\s*([0-9.]+)\s*mm\s*\(([0-9.]+)cm3\)\s*|\A\s*;\s*filament used\s*\[mm\]\s*=+\s*([0-9.]+)|\A\s*;\s*filament used\s*\[cm3\]\s*=+\s*([0-9.]+)', gcode_line)
```

Comment:
Is this really the output of prusa slic3r now?  Do you have a sample file that I could see?  Thanks!

Reply:
i uploaded the examples:
https://github.com/HTXF-g/example-files-etc.../tree/main/Example-files/2023/05